File: demo/app/src/main/java/com/jst/compinit/demo/MyApplication.java
package com.jst.compinit.demo;

import android.app.Application;

import com.jst.compinit.ComponentInitializer;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        ComponentInitializer.setDebug(true);
        ComponentInitializer.initComponents(this);
    }
}

File: demo/app/src/main/java/com/jst/compinit/demo/MyApplication.kt
package com.jst.compinit.demo

import android.app.Application
import com.jst.compinit.ComponentInitializer

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        ComponentInitializer.setDebug(true)
        ComponentInitializer.initComponents(this)
    }
}